Sources/ProjectSpec/TargetSource.swift

```diff
@@ -40,6 +40,11 @@ public struct TargetSource: Equatable {
         case carbonResources
         
         public struct CopyFilesSettings: Equatable, Hashable {
+            public static let xpcServices = CopyFilesSettings(
```

Comment:
I like this. We could add a similar static `modulemap` property.
Could we move it into `XcodeGenKit` as an extension though?

Reply:
The main reason I did this one here, is that it shows up in the dropdown for a destination in Xcode, and if you these exact settings Xcode will render it as such.

The `modulemap` based one is more a convention, and that I would put into an extension in `XcodeGenKit`.